content/webapp/components/Contributors/Contributor.tsx

```diff
@@ -31,13 +31,13 @@ const Contributor = ({ contributor, role, description }: ContributorType) => {
                   transform: 'rotateZ(6deg) scale(1.2)',
                 }}
               >
-                <PrismicImage image={contributor.image} />
+                <PrismicImage image={contributor.image} maxWidth={72} />
```

Comment:
Wasn't this in a different PR?

Reply:
I spy a merge conflict, rebasing should fix.